forest/bonsai/simulate_gps_data.py

```diff
@@ -604,21 +601,10 @@ def calculate_trip(self, origin: Tuple[float, float],
         if coords_str in self.trips.keys():
             path = self.trips[coords_str]
         else:
-            for try_no in range(3):
-                try:
-                    path, _ = get_path(
-                        origin, destination, transport, api_key
-                        )
-                except RuntimeError:
-                    if try_no == 2:
-                        raise
-                    else:
-                        time.sleep(30)
-                        continue
-                else:
-                    path = get_basic_path(path, transport)
-                    self.trips[coords_str] = path
-                    break
+            time.sleep(1.5)
```

Comment:
It turns out that it's possible to use a local ORS instance that would have no rate limit, so this timeout value should be not be hardcoded but sourced from an environment variable for example. Also, it should be possible to set the ORS client's `base_url` the same way.
https://github.com/GIScience/openrouteservice-py#local-ors-instance
https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5

Reply:
So you propose I put in the code something like `ors_sleep_time = os.environ.get('ORS_TIME_BETWEEN_CALLS')` and the user of bonsai would have set that environment before? and for the base url the same thing?

Reply:
Yes but add a prefix to the env var name for consistency and to avoid potential name collisions, a default value, and a comment explaining how and why this magic number was derived, something like:
```
ORS_API_DELAY = os.getenv("FOREST_ORS_API_DELAY", default=1.5)
ORS_API_BASE_URL = os.getenv("FOREST_ORS_BASE_URL", default="https://...")
```
It's probably a good idea to move them to a separate file like `forest/forest/constants.py` along with `OVERPASS_URL` (might be a good idea to source that from an env var too).